src/scripts/build_synthetic_nts.py

```diff
@@ -0,0 +1,87 @@
+"""Script that builds a synthetic vocie audio from reading 
+the nst dataset. 
+
+Usage:
+    python build_syntethic_nts.py 
+"""
+
+import os
+import subprocess
+
+from gtts import gTTS
+import pandas as pd
+
+
+def generate_speech_mac(text, filename):
```

Comment:
Missing type hints

Reply:
yes, I have to start on this. I assume with newer version of python, it runs faster and are less error prone with type hints.

Reply:
Yep, and in general it's less error prone, so it will save both us and others a bunch of time when doing that. For instance, our MyPy checks uses these during the pre-commit hook. Also, linting and documentation tools automatically use the type hints as well.